spec/system/bitklavier_single_item_metadata_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+describe 'PDC Describe Bitklavier Single item page', type: :system, js: true do
+  let(:rss_feed) { File.read(File.join(fixture_path, 'pdc_describe', 'works.rss')) }
+  let(:resource1) { File.read(File.join(fixture_path, 'pdc_describe', 'resource1.json')) }
+  let(:bitklavier_binaural_json) { File.read(File.join(fixture_path, 'pdc_describe', 'bitklavier_binaural.json')) }
+  let(:rss_url_string) { "https://pdc-describe-prod.princeton.edu/describe/works.rss" }
+  let(:indexer) { DescribeIndexer.new(rss_url: rss_url_string) }
+
+  before do
+    Blacklight.default_index.connection.delete_by_query("*:*")
+    Blacklight.default_index.connection.commit
+    stub_request(:get, "https://pdc-describe-prod.princeton.edu/describe/works.rss")
+      .to_return(status: 200, body: rss_feed, headers: {})
+    stub_request(:get, "https://pdc-describe-prod.princeton.edu/describe/works/6.json")
+      .to_return(status: 200, body: resource1, headers: {})
+    stub_request(:get, "https://pdc-describe-prod.princeton.edu/describe/works/20.json")
+      .to_return(status: 200, body: bitklavier_binaural_json, headers: {})
+    indexer.index
+  end
+
+  it "has expected header fields" do
+    visit '/catalog/doi-10-34770-r75s-9j74'
+    expect(page).to have_css '.document-title-heading'
+    expect(page).to have_css '.authors-heading'
+    expect(page).to have_css 'div.authors-heading > span > i.bi-person-fill'
+    expect(page).to have_css '.issue-date-heading'
+  end
+
+  # rubocop:disable RSpec/ExampleLength
+  it "has expected metadata" do
+    visit '/catalog/doi-10-34770-r75s-9j74'
+    expect(page).to have_content "bitKlavier Grand Sample Library—Binaural Mic Image"
+    authors = "<span>Trueman, Daniel; Wang, Matthew; Villalta, Andrés; Chou, Katie; Ayres, Christien</span>"
+    expected_values = [authors]
+    expected_values.each do |value|
```

Comment:
This is a very strange construct for checking just the authors. If you are expecting to check the metadata fields, then I would expect title to be a part of this loop, but instead it is checked separately.

Reply:
Yeah, you're not wrong that it's awkward. I copied it from what we did for DataSpace indexing (https://github.com/pulibrary/pdc_discovery/blob/main/spec/system/single_item_metadata_spec.rb#L24-L35). Basically I just copied the whole spec because I thought that would be an easy way to keep track of when we've reached feature parity.